zipkin-example/src/main/scala/com/twitter/zipkin/example/Main.scala

```diff
@@ -1,20 +1,18 @@
 package com.twitter.mycollector
 
-import com.twitter.zipkin.conversions.thrift._
-import com.twitter.finagle.Http
-import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle.Httpx
 import com.twitter.server.{Closer, TwitterServer}
-import com.twitter.util.{Await, Closable, Future}
+import com.twitter.util.{Await, Closable}
 import com.twitter.zipkin.anormdb.AnormDBSpanStoreFactory
-import com.twitter.zipkin.collector.SpanReceiver
 import com.twitter.zipkin.common.Span
-import com.twitter.zipkin.{thriftscala => thrift}
```

Comment:
sorry about the import change.. the amount of unused ones was driving me nuts. I don't feel like making a pull request just to do that.